packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -140,45 +137,114 @@ export default function ExperimentHeader({
     }
   }
   const canRunExperiment = canEditExperiment && hasRunExperimentsPermission;
+  const hasVerifiedConnection =
+    checklistItemsRemaining !== null && checklistItemsRemaining > 0;
 
   const isUsingHealthUnsupportDatasource =
     !dataSource || datasourcesWithoutHealthData.has(dataSource.type);
   const disableHealthTab = isUsingHealthUnsupportDatasource || !!dimension;
 
+  async function startExperiment() {
+    startCelebration();
+    if (!experiment.phases?.length) {
+      if (newPhase) {
+        newPhase();
+        return;
+      } else {
+        throw new Error("You do not have permission to start this experiment");
+      }
+    }
+
+    await apiCall(`/experiment/${experiment.id}/status`, {
+      method: "POST",
+      body: JSON.stringify({
+        status: "running",
+      }),
+    });
+    await mutate();
+    track("Start experiment", {
+      source: "experiment-start-banner",
+      action: "main CTA",
+    });
+    setTab("results");
+    setShowStartExperiment(false);
+  }
+
   return (
     <>
       <div className="experiment-header bg-white px-3 pt-3">
-        {startExperiment && experiment.status === "draft" && (
+        {showStartExperiment && experiment.status === "draft" && (
           <Modal
             open={true}
-            size="lg"
-            close={() => setStartExperiment(false)}
+            size="md"
+            closeCta={
+              hasVerifiedConnection ? (
+                <button
+                  className="btn btn-primary"
+                  onClick={() => setShowStartExperiment(false)}
+                >
+                  Close
+                </button>
+              ) : (
+                // This is a bit odd, but design requested we use the closeCTA as an override in this case
```

Comment:
Ticket for this [here](https://www.notion.so/growthbook/UI-Issues-to-Work-into-Releases-14ed60043f2f4b1cae74db9ec57afec8?p=c31d55afe8424e04827092c249d10941&pm=s).